content/tutorial-escrow.md

```diff
@@ -537,6 +536,11 @@ _Websocket_
 
 <!-- MULTICODE_BLOCK_END -->
 
+
+The response will look something like the following example. Note that the response includes all pending escrow objects with `rfztBskAVszuS3s5Kq7zDS74QtHrw893fm` as the sender or receiver address, where the sender address is the `Account` value and the receiver address is the `Destination` value.
+
+In this case, the second and fourth escrow objects have `rfztBskAVszuS3s5Kq7zDS74QtHrw893fm` as their sender address (`Account`) value and are what we are looking for.
```

Comment:
Style nitpick: we avoid phrases like "what we are looking for". Suggested rephrase:

> In this example, `rfztBskAVszuS3s5Kq7zDS74QtHrw893fm` is the the `Account` (sender) address in the second and fourth escrow objects.

(To be honest, my phrasing here still feels awkward, so feel free to tweak some more.)

Reply:
Yep - got it - thank you! 👍  Just as an FYI, I went with this general sentence structure for sender and receiver:

> In this example, the second and fourth escrow objects meet our lookup criteria because their `Account` (sender address) values are set to `rfztBskAVszuS3s5Kq7zDS74QtHrw893fm`.